rts/System/Platform/Linux/Futex.cpp

```diff
@@ -29,7 +33,11 @@ void spring_futex::lock()
 
 	do {
 		if ((c == 2) || __sync_val_compare_and_swap(&mtx, 1, 2) != 0)
+#ifndef __OpenBSD__
 			syscall(SYS_futex, &mtx, FUTEX_WAIT_PRIVATE, 2, NULL, NULL, 0);
+#else
+			futex(&mtx, FUTEX_WAIT, 2, NULL, NULL);
```

Comment:
not _PRIVATE on the FIUTEX constant?

Reply:
I didn't use `_PRIVATE` because it's not documented in our man page, but I see that these constants are also defined in `sys/futex.h`. I'll do a build with the _PRIVATE parts and if that works, I'll change it!

Reply:
changed it to `_PRIVATE` futex flags